aws-iam-role-admin/main.tf

```diff
@@ -0,0 +1,182 @@
+module "role" {
+  source             = "../aws-iam-role-crossacct"
+  role_name          = var.role_name
+  iam_path           = var.iam_path
+  source_account_ids = var.source_account_ids
+  saml_idp_arns      = var.saml_idp_arns
+  oidc               = var.oidc
+
+  project = var.project
+  env     = var.env
+  service = var.service
+  owner   = var.owner
+}
+
+resource "aws_iam_role_policy_attachment" "readonly" {
+  count      = var.readonly ? 1 : 0
+  role       = module.role.role_name
+  policy_arn = "arn:aws:iam::aws:policy/ReadOnlyAccess"
+}
+
+resource "aws_iam_role_policy_attachment" "all" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "aws" in "required_providers"